src/main/java/racingcar/model/Cars.java

```diff
@@ -0,0 +1,47 @@
+package racingcar.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import racingcar.Main;
+import racingcar.util.Input;
+
+public class Cars {
+	private Map<Integer, Car> cars;
+
+	public Cars() {
+		cars = new HashMap<>();
+	}
+
+	public int size() {
+		return cars.size();
+	}
+
+	public void createCar(Integer num, Car car) {
+		cars.put(num, car);
+	}
+
+	public Car car(Integer num) {
+		return cars.get(num);
+	}
+
+	public Map<Integer, Car> cars() {
+		return cars;
+	}
+
+	public void createCars(String[] carNames) {
```

Comment:
```suggestion
	public static Map<Integer, Car> createCars(String[] carNames) {
```

또는 
```suggestion
	public static Cars createCars(String[] carNames) {
```

위와 같이 create 의미를 생성한 후 반환하는 구조로 고민해 보면 좋겠다.